darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -140,17 +140,13 @@ def fit(
         self.models = [model.untrained_model() for model in self.models]
 
         for model in self.models:
+            kwargs = dict(series=series)
             if self.is_global_ensemble:
```

Comment:
couldn't we just fully ignore the self.is_global_ensemble and do the following for any model:

```
kwargs = dict(series=series)
if model.supports_past_covariates:
    kwargs["past_covariates"] = past_covariates
if model.supports_future_covariates:
    kwargs["future_covariates"] = future_covariates
model.fit(**kwargs)